src/main/java/apache/ranger/client/model/Role.java

```diff
@@ -0,0 +1,45 @@
+package apache.ranger.client.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Shlpeng
+ */
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Role {
+    private int id;
+    private boolean isEnabled;
+    private String createdBy;
+    private String updatedBy;
+    private String name;
+    private String description;
+    private Map<String, Object> options;
+    private List<RoleUser> users;
+    private List<Object> groups;
```

Comment:
Create valid model instead of object.

Reply:
done， please check it

from apache ranger：
<img width="908" alt="image" src="https://github.com/user-attachments/assets/55a5c632-1c01-429e-acad-8be313df06fd">